### ISSUE TITLE
Save Inject->Head to file instead SharedPreferences

### ISSUE THREAD

**Issue by gesu92** [gsantner/markor#2352]
Hello, first of all, love this app, I think that I use almost for everything, now I been playing with the js and css head inject option, and really be nice have that codes on 2 differents files, and in a place that can be synced, because the current editor or viewer for that it's really hard to manage with biggers codes, hope you can help me with this, thanks for all.

pd= sorry my bad english.

**Comment by gsantner** [comment]
Your question is if what you enter at the inject option, could be stored to a file in .app , instead of application settings.

Is that right?

**Comment by gesu92** [comment]
No sir, I say instead of having the css stylesheet and the js in the settings, have those codes in 2 different files, in a location that can be accesible for the users.
example:
.app/js/markor-headinject.js
.app/css/markor-headinject.css

**Comment by EricMFink** [comment]
You can save your css as a file & use a stylesheet link in the "inject -> head" setting, e.g.:

```
<link rel="stylesheet" type="text/css" href="/storage/emulated/0/Markor/Templates/markor.css" >

```

**Comment by gesu92** [comment]
yeah, that works fine, and the `<script type="text/javascript" src="..."></script> ` works fine too, when I open this issue that don't works, but I think that only put wrong the source path, thanks.
